src/components/Grid/Column.tsx

```diff
@@ -0,0 +1,60 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import { rivetize } from '../Rivet';
+
+type ColumnWidth = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+type AutoColumnWidth = ColumnWidth | boolean;
+
+interface ColProps {
+    last?: boolean;
+    lg?: AutoColumnWidth,
```

Comment:
There are a couple fields here that have comma line-endings instead of semicolons.